src/app/page.tsx

```diff
@@ -2,15 +2,20 @@
 
 import { Button } from '@/components/button'
 import { Heading } from '@/components/heading'
-import { murmurClient } from './murmurClient'
+import { masterService, murmurClient } from './murmurClient'
 import { useState } from 'react'
 import { Input } from '@/components/input'
+import { Extrinsic } from '../../../murmur.js/src/types'
```

Comment:
Also to update https://github.com/ideal-lab5/murmur-dapp/pull/22/files#r1795960618